src/ape_console/_cli.py

```diff
@@ -110,11 +109,8 @@ def console(project=None, verbose=None, extra_locals=None):
             project_path=project.path,
         )
 
-        try:
+        if not environ.get("APE_TESTING"):
             faulthandler.enable()  # NOTE: In case we segfault
-        except io.UnsupportedOperation:
-            # Likely running in tests
-            pass
```

Comment:
This seemed to only arise from running the test suite, so I opted to change up enabling the fault handler only when we aren't testing (i.e. have the test env set)